spec/API_specification/linear_algebra_functions.md

```diff
@@ -265,9 +265,30 @@ TODO
 TODO
 
 (function-slogdet)=
-### slogdet()
+### slogdet(x, /)
 
-TODO
+Returns the sign and the natural logarithm of the absolute value of the determinant of a square matrix (or stack of square matrices) `x`.
+
+```{note}
+
+The purpose of this function is to calculate the determinant more accurately when the determinant is either very small or very large, as calling `det` may overflow or underflow.
+```
+
+#### Parameters
+
+-   **x**: _&lt;array&gt;_
+
+    -   input array having shape `(..., M, M)` and whose innermost two dimensions form square matrices. Must have a data type of either `float32` or `float64`.
+
+#### Returns
+
+-   **out**: _Tuple\[ &lt;array&gt;, ... ]_
+
+    -   if `x` is a two-dimensional array, a namedtuple (`sign`, `logabsdet`) whose first element must be zero-dimensional array containing a number representing the sign of the determinant and whose second element is a zero-dimensional array containing the determinant. Otherwise, a namedtuple (`sign`, `logabsdet`) whose first element must be a non-zero dimensional array containing the sign for each square matrix and whose second element must be a non-zero dimensional array containing the determinant for each square matrix.
```

Comment:
Can we simplify the description here a bit to avoid repeating? Maybe we can just say `sign` and `logabsdet` are arrays of shape `x.shape[:-2]`?

Reply:
Addressed.